vctl/command/print.go

```diff
@@ -106,3 +106,7 @@ func (cmd *Command) printFrontend(f *engine.Frontend, ms []engine.Middleware) {
 func writeS(w io.Writer, v string) {
 	w.Write([]byte(v))
 }
+
+func PrintError(cmd *Command, err error) {
+	cmd.printError(err)
```

Comment:
Just capitalize `printError` and use it in `vctl/main.go` there is no point in introducing an exported wrapper.